### PR TITLE
added chatbot feature for 1 reply only

### PR DESCRIPTION
**Pull request by kaartav**
IF we hit the url, the  history will start from zero .Have to see do we send prompts from user without refreshing the url

### CODE REVIEW

**Review by rohitkotha10 — APPROVED** [review]

**Review by rohitkotha10 — CHANGES_REQUESTED** [review]
Format files and fix imports

File: users/Post_stream_chatbot.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import json
+import requests
```
Comment (rohitkotha10) [inline users/Post_stream_chatbot.py:2]: refer this: https://github.com/WHEUDAIMONIA/EudaimoniaBackend//blob/main/app.py#L1

File: app.py [hunk 2]
```diff
@@ -35,5 +36,16 @@ def add_user():
     return response
 
 
+@app.route("/chatbot", methods=["POST"])
+def chatbot():
+    prompt = request.form.get("prompt")
+    conversation_history = [
+        {"role": "system", "content": "Your name is Yuki. You are a calm and soothing therapist."},
+    ]
+    url = os.getenv("TARGET_URL")
```
Comment (rohitkotha10) [inline app.py:45]: please use a better name. for URL, can use it directly. Need to hide the Key only

**Review by rohitkotha10 — APPROVED** [review]

**Comment by kaartav** [comment]
Send  a header
key:Content-Type
value:application/x-www-form-urlencoded
Body
key:prompt
value: Your prompt

**Pull request merged**